src/Microsoft.ServiceHub.Framework/ServiceBrokerAggregator.cs

```diff
@@ -221,15 +221,22 @@ public ForceMarshalingBroker(IServiceBroker inner)
 		public async ValueTask<T?> GetProxyAsync<T>(ServiceRpcDescriptor serviceDescriptor, ServiceActivationOptions options, CancellationToken cancellationToken)
 			where T : class
 		{
-			T? client = await this.inner.GetProxyAsync<T>(serviceDescriptor, options, cancellationToken).ConfigureAwait(false);
-			if (client == null)
+			IDuplexPipe? pipe = await this.inner.GetPipeAsync(serviceDescriptor.Moniker, options, cancellationToken).ConfigureAwait(false); ;
```

Comment:
you have double-semicolons at the end of this line.

Reply:
I had warnings hidden in VS for an earlier solution so didn't realize it. Fixed in the last iteration.